docs/basics/data-center-endpoints.md

```diff
@@ -85,7 +85,7 @@ URLs vary based on the services you're using, your data center, and the types of
 <td>*.miso.saucelabs.com</td><td><code>66.85.48.0/21</code></td>
 </tr>
 <tr>
-<td>Other endpoints</td><td><code>162.222.72.0/21</code><br/><code>66.85.48.0/21</code></td>
+<td>Other endpoints</td><td><code>34.125.90.96/27</code><br/><code>162.222.72.0/21</code><br/><code>66.85.48.0/21</code></td>
```

Comment:
context 

> ondemand.us-west-1.sacuelabs.com has address 185.53.179.171
> ondemand.sacuelabs.com has address 185.53.179.171
> jpmc.ondemand.saucelabs.com has address 66.85.49.22

`66.85.*` and `185.53.*` will be deprecated (EOL date??) and replaced by the new IP block added

Reply:
I believe that DNS cutover will happen on 10th October